data/remnant missions.txt

```diff
@@ -649,66 +644,135 @@ mission "Remnant: Catalytic Ramscoop"
 			choice
-				`	"Yes, you are welcome to examine it."`
-					goto deep
-				`	"Yes, but don't touch anything."`
+				`	"Sure, you may examine it."`
+					goto examine
 				`	"No, stay away from my ship."`
 					decline
 
-			`	"Don't worry," he sings with a tinge of reproach. "I am a Remnant, not a Korath."`
-			`	He proceeds to methodically examine your ship from every angle on the ground, eventually focusing on your weapon ports and hardpoints. After examining the <ship> for quite some time, he walks over to you.`
-				goto more
-
-			label deep
-			`	He clips his harness to one of the overhead lifts, presses a button on the back of his gauntlet, and is promptly whisked up into the air. The lift appears designed to give engineers and technicians almost as much freedom as a zero-g environment by suspending them with just enough force to counteract gravity, so they can move around a ship with ease. Having completed a quick once-over he quickly gravitates to the weapon hardpoints. After examining the <ship> for quite some time, he calls down to you.`
-
-			label more
-			`	"I have never seen weapons designed to handle such a high heat output! Is this some kind of plasma based weapon?" he asks, gesturing at a plasma cannon. You nod, and he continues, "Intriguing. We have noticed that the Korath ships seem to run particularly hot, and have speculated that overheating them might be an effective means of disabling them. I would enjoy researching these a bit more. If you could acquire some for us you would be well compensated."`
+			label examine
+			`	He clips his harness to one of the overhead lifts, presses a button on the back of his gauntlet, and is promptly whisked up into the air. The lift appears designed to give engineers and technicians almost as much freedom as a zero-g environment by suspending them with just enough force to counteract gravity, so they can move around a ship with ease. He appears to have already identified the weapon hardpoints, and heads straight to them.`
+			`	"I have never seen weapons designed to handle such a high heat output! Is this some kind of plasma based weapon?" he asks, gesturing at it. You nod, and he continues, "Impressive. Our historical records mention that such weapons were just starting to be used by military forces at the time of our flight. Such a use for plasma had slipped from memory until we noticed that the Korath ships seem to run particularly hot. Our tacticians have speculated that overheating them might be an effective means of disabling them." He pauses, then chants again "I would enjoy researching these a bit more."`
+			branch
+				has "FW Plasma Testing 1B: done"
+			`	You recall that you helped Kraz Cybernetics test some experimental plasma turrets for the Free Worlds."`
```

Comment:
Again, this branch isn't leading anywhere, and I don't see it as necessary anyways.